design-guidelines/page-layout-patterns.md

```diff
@@ -0,0 +1,145 @@
+
+![contentAction-txtFilter+filter+actions](./img/page-layout-patterns/contentAction-txtFilter+filter+actions.png)
+
+#### Examples
+
+##### Standard layout
+
+![standard-example-basic](./img/page-layout-patterns/standard-example-basic.png)
+
+![standard-example-withActions](./img/page-layout-patterns/standard-example-withActions.png)
+
+![standard-example-subheaderSummary](./img/page-layout-patterns/standard-example-subheaderSummary.png)
+
+![standard-example-subheaderPromobanner](./img/page-layout-patterns/standard-example-subheaderPromobanner.png)
+
+![standard-example-tabs](./img/page-layout-patterns/standard-example-tabs.png)
+
+##### Primary content actions
+
+![contentAction-example-withAction](./img/page-layout-patterns/contentAction-example-withAction.png)
+
+![contentAction-example-withTxtFilter](./img/page-layout-patterns/contentAction-example-withTxtFilter.png)
+
+![contentAction-example-withTxtFilter+filter+actions](./img/page-layout-patterns/contentAction-example-withTxtFilter+filter+actions.png)
+
+### Editorial layout
+
+The editorial layout is just like the standard layout. The only differences are that the primary content is constrained to a maximum width, and the primary content doesn't typically have an action area. The maximum width is helpful for maintaining readibility for large blocks of text.
+
+![editorial](./img/page-layout-patterns/editorial.png)
+
+#### Examples
+
+![editorial-example](./img/page-layout-patterns/editorial-example.png)
+
+### Path selection layout
+
+The path selection layout is used when users need to make a selection to determine where they will be taken to complete their task.
+
+The app header bar and app sidebar are hidden when users need to focus on choosing the appropriate path. A simple header may be used to show very basic info (e.g.: page title) and a button to cancel out of the flow.
+
+![pathSelection](./img/page-layout-patterns/pathSelection.png)
+
+#### Examples
+
+![pathSelection-example-one](./img/page-layout-patterns/pathSelection-example-one.png)
+
+![pathSelection-example-two](./img/page-layout-patterns/pathSelection-example-two.png)
+
+### Tabbed form layout
+
+The tabbed form layout is used to display forms with multiple complex sections.
+
+The app header bar and app sidebar are hidden when using this layout to allow users to focus on the complex form. A simple header may be used to show: very basic info (e.g.: page title), a button to cancel out of the form, and a button to submit the form.
+
+![tabbedForm](./img/page-layout-patterns/tabbedForm.png)
+
+#### Examples
+
+![tabbedForm-example](./img/page-layout-patterns/tabbedForm-example.png)
+
+### Simple focused layout
+
+The simple focused layout is used to present users with a simple task that needs to be completed before moving on to their next step in their session.
+
+![simpleFocused](./img/page-layout-patterns/simpleFocused.png)
```

Comment:
Hmmmm.... what use case do we have for such with just 1 card? 

Conductor upload CSV design has somewhat of this design (Can talk to Nan but there is case where I think use can just upload by CSV and doesn't have the first step to choose?)

Reply:
Login screens. This would have been more clear if I added the correct example images 😅